source/user-manual/reference/ossec-conf/global.rst

```diff
@@ -441,6 +443,50 @@ Example:
 
   <queue_size>16384</queue_size>
 
+.. _reference_agents_disconnection_time:
+
+agents_disconnection_time
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 4.1.0
+
+This sets the time since the last keepalive of an agent to consider it as disconnected.
```

Comment:
Let's reword a little bit this sentence...

```
This sets the time after which the manager considers an agent as disconnected since its last keepalive.
```

Reply:
Replaced!